packages/backend/services/db/dbFirebase.js

```diff
@@ -90,6 +90,20 @@ const findAllBatches = async () => {
   return batchesSnapshot.docs.map(doc => ({ id: doc.id, ...doc.data() }));
 };
 
+const findLatestOpenBatch = async () => {
+  const batchesSnapshot = await database
+    .collection("batches")
+    .where("status", "==", "open")
+    .orderBy("startDate", "desc")
+    .limit(1)
+    .get();
```

Comment:
Oh, this broke the site a bit :(

We talked about this, but forgot to double-check on the live site. These kind of queries (more complex than a simple where: e.g. where + orderby) requires an index that needs to be created manually.

Just created it and everything looks great!